src/components/featureItem.tsx

```diff
@@ -0,0 +1,20 @@
+interface Props {
+  children: string;
+  title: string;
+  src: string;
+  order?: string;
+}
+
+export const FeatureItem = ({ children, title, src, order }: Props) => {
+  return (
+    <div
+      className={`flex ${order} justify-around items-center mb-20 text-xl font-medium text-yellow-900`}
```

Comment:
You can replace `order` with `className`. Also, use the example from the comment on the `button.ts` file:

[Watch video](https://youtu.be/re2JFITR7TI?si=o0SkTTuWJRRRVgeB)

```suggestion
      className={cn("flex justify-around items-center mb-20 text-xl font-medium text-yellow-900", className)}
```